src/Lib/2D/Surface.hpp

```diff
@@ -18,7 +18,7 @@ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 #ifndef __Surface_hpp__
 #define __Surface_hpp__
 
-#include <SDL2/SDL.h>
+#include <SDL.h>
```

Comment:
This is worse - we should be namespacing our dependencies on include.

Reply:
I'm not sure how to handle that. On Slackware, Ubuntu, Manjaro, Debian, `sdl2-config --cflags` returns `# sdl2-config --cflags
-I/usr/include/SDL2 -D_REENTRANT` so the compiler is going to look in SDL2/, therefore, `#include <SDL.h>` is all that's needed in that case. Build systems typically call `sdl2-config` to get the right flags for programs requiring sdl2.

When the source is download in the case of fallbacks with meson, the header is in 'subprojects/SDL2-2.28.5/include'

Reply:
Okay! I guess it's fine for now until it's an issue, don't want to cause extra work if it's a lot of trouble.

Reply:
my thoughts too. If I knew a better way, I'd either do it, or I'd take a break and do it. ;) But definitely we'll have to change that if it becomes an issue. Note it's passing all CI tests (the Cirrus failure is unrelated).

Reply:
> This is worse - we should be namespacing our dependencies on include.

This is an illegal maneuver. The `#include` name of a dependency is exactly no more and no less than whatever the dependency tells you to use.

There is no such concept as namespacing. The "namespacing" here is that the sdl project has moral rights to the name SDL.h, and specified that if your application uses version 2 then you add the SDL2/ directory to your include paths and use `#include <SDL.h>`, and if you use version 3 then you add the root includedir to your include paths and use `#include <SDL3/SDL.h>`.

Yes, the style changed between versions. But even if you have both sdl 2 and 3 brought into your project at the same time (???) version 3's headers won't be available as `<SDL.h>` so there is no clash.